gdc_client/defaults.py

```diff
@@ -34,3 +34,6 @@
 proxy_port = 9000
 
 HTTP_CHUNK_SIZE = 1024 * 1024 * 1024  # 1 GiB
+
+# Bringing this back to avoid errors in other modules
+part_size = HTTP_CHUNK_SIZE
```

Comment:
Where is uses this?

Reply:
Just double checked, seems to be nowhere else.